javascript/fluent.js

```diff
@@ -125,16 +125,29 @@
 				} else {
 					label.attr('title', 'Using default locale value');
 				}
+			},
+			/**
+			 * The JSON in the data attribute will be encoded, so this is a bit of a hack to decode it before parsing
```

Comment:
@tractorcow let me run a quick test without JSON encoding and see - I guess it should work the same way :)